tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -85,6 +85,25 @@ def load_conditions(session):
 
     return conditions_list
 
+def read_asic_name(hwsku):
```

Comment:
Add some comments here as documentation?

Also update the README.md file for this plugin?

Reply:
done